src/Server/Configuration.php

```diff
@@ -440,26 +440,4 @@ public function getPty()
     {
         return $this->pty;
     }
-
-    /**
-     * Set pty for ssh2 connection. For retro compatibility
-     *
-     * @param $ssh2Pty
-     * @deprecated
-     */
-    public function setSsh2Pty($ssh2Pty)
-    {
-        $this->setPty($ssh2Pty);
-    }
-
-    /**
-     * Get pty option for ssh2 connection. For retro compatibility
-     *
-     * @deprecated
```

Comment:
@elfet @vladdnepr Do we still need this deprecation?

Meanwhile, @elfet is there any rule about setting and using `@deprection` tag? How long does it live? Should we set a version when it will be removed?